alacritty/src/display.rs

```diff
@@ -471,7 +473,18 @@ impl Display {
 
             self.renderer.with_api(&config.ui_config, config.cursor, &size_info, |mut api| {
                 // Iterate over all non-empty cells in the grid.
-                for cell in grid_cells {
+                for mut cell in grid_cells {
+                    // Invert the active match in vi-less search.
```

Comment:
Out of curiosity, do you have selection configured to invert too?

Reply:
No, actually, but I don't remember why `No`, since I sometimes inver, sometimes don't. You can't use selection as a fallback, since it could confuse users.